doc/release/v3_2_0.md

```diff
@@ -38,15 +38,21 @@ New Features
   - `utils::horzSplit`
   - `utils::vertConcat`
   - `utils::horzConcat`
+* Added the following methods to `VectorOf`:
+  - VectorOf(std::initializer_list<T>)
+  - begin(), end(), cbegin(), cend()
+  - reserve(size_t)
+  - capacity()
+  - push_back(T&&)
+  - emplace_back(_Args...)
+* Deprecated `VectorOf::getFirst()` in favour of `data()` and `begin()`. See
+  deprecation message to replace it properly in your code.
```

Comment:
I would repeat the deprecation message in the release notes, since this is something "permanent"